dvclive/live.py

```diff
@@ -191,11 +203,13 @@ def make_summary(self):
             json.dump(summary_data, f, indent=4, cls=NumpyEncoder)
 
     def make_report(self):
-        if self._report == "html":
-            html_report(self.dir, self.summary_path, self.html_path)
+        if self._report is not None:
```

Comment:
@dberenbaum 

> Does the markdown report get generated automatically on live.next_step()? 

Yes.

> Do we need to follow up on https://github.com/iterative/dvclive/issues/231 to clarify behavior in this scenario now that we have non-html reports?

Not sure I follow what's the difference of considering non-html report in the discussion